forum/users/models.py

```diff
@@ -1,3 +1,65 @@
 from django.db import models
 
-# Create your models here.
+from django.db import models
+from django.contrib.auth.models import (
+    AbstractBaseUser,
+    BaseUserManager,
+    PermissionsMixin,
+)
+import datetime
+import re
+
+ROLE_CHOICES = [
+    (0, "Unassigned"),
+    (1, "Startup"),
+    (2, "Investor"),
+]
+
+
+class CustomUserManager(BaseUserManager):
+    def create_user(self, email, password=None, **extra_fields):
+        if email:
+            email_pattern = r"^[a-zA-Z0-9]+@[a-zA-Z0-9]+\.[a-zA-Z0-9]+$"
```

Comment:
The regex for email validation is simplistic and might not cover edge cases.
You may use Django's built-in email validation:
```
from django.core.validators import validate_email
from django.core.exceptions import ValidationError

try:
    validate_email(email)
except ValidationError:
    raise ValueError("The Email is not valid")
```